utils.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import "unicode"
+
+func ToSnake(in string) string {
```

Comment:
We probably want them to exist next to the code, not just out in space, to prevent regressions